libs/ui/elements/src/lib/metadata-contact/metadata-contact.component.html

```diff
@@ -1,32 +1,94 @@
-<div *ngIf="shownOrganization" class="mb-6 sm:mb-12">
-  <p class="text-gray-700 text-xs mb-3 uppercase" translate>
-    record.metadata.contact
-  </p>
-  <div class="flex mb-1">
+<div class="py-4 px-5 rounded bg-gray-100 text-black">
+  <div *ngIf="shownOrganization" class="mb-6 sm:mb-12">
+    <p class="text-sm mb-3 font-medium" translate>record.metadata.contact</p>
     <div
-      class="text-primary font-title text-21 mr-2 cursor-pointer hover:underline"
-      (click)="onOrganizationClick()"
+      *ngIf="contacts[0].organization?.logoUrl?.href"
+      class="mb-3 flex items-center justify-center border-solid border border-gray-300 rounded-md bg-white h-32"
     >
-      {{ shownOrganization.name }}
+      <gn-ui-thumbnail
+        class="relative h-full w-full"
+        [thumbnailUrl]="contacts[0].organization.logoUrl.href"
+        fit="contain"
+      ></gn-ui-thumbnail>
+    </div>
+    <div class="flex mb-1">
+      <div
+        class="text-primary font-title text-21 mr-2 cursor-pointer hover:underline"
+        (click)="onOrganizationClick()"
+        data-cy="organization-name"
+      >
+        {{ shownOrganization.name }}
+      </div>
+    </div>
+    <div *ngIf="shownOrganization.website" class="mb-6">
+      <a
+        [href]="shownOrganization.website"
+        target="_blank"
+        class="contact-website text-primary text-sm cursor-pointer hover:underline transition-all"
+        >{{ shownOrganization.website }}
+        <mat-icon
+          class="material-symbols-outlined !w-[12px] !h-[12px] !text-[12px] opacity-75 shrink-0"
+          >open_in_new</mat-icon
+        >
+      </a>
+    </div>
+  </div>
+  <div *ngIf="contacts[0].phone" class="mb-5">
```

Comment:
```suggestion
  <div *ngIf="contacts[0]?.phone" class="mb-5">
```

Otherwise JS errors pop in case of a missing contact